v20.2/stream-data-out-of-cockroachdb-using-changefeeds.md

```diff
@@ -12,8 +12,14 @@ While CockroachDB is an excellent system of record, it also needs to coexist wit
 
 The main feature of CDC is the changefeed, which targets an allowlist of tables, called the "watched rows". There are two implementations of changefeeds:
 
-- [Core changefeeds](#create-a-core-changefeed), which stream row-level changes to the client indefinitely until the underlying connection is closed or the changefeed is canceled.
-- [Enterprise changefeeds](#configure-a-changefeed-enterprise), where every change to a watched row is emitted as a record in a configurable format (`JSON` or Avro) to a configurable sink ([Kafka](https://kafka.apache.org/)).
+| [Core changefeeds](#create-a-core-changefeed)   | [Enterprise changefeeds](#configure-a-changefeed-enterprise) |
+--------------------------------------------------|-----------------------------------------------------------------|
+| Useful for prototyping or quick testing. | Recommended for production use. |
+| Available in CockroachDB and CockroachCloud Free Tier. | Available in CockroachCloud or with an [enterprise license](enterprise-licensing.html) in CockroachDB. |
+| Streams indefinitely until underlying SQL connection is closed. | Maintains connection to configured sink. |
+| Create with [`EXPERIMENTAL CHANGEFEED FOR`](changefeed-for.html). | Create with [`CREATE CHANGEFEED`](create-changefeed.html). |
+| Watches one or multiple tables in a comma-separated list. | Emits every change to a "watched" row as a record in a configurable format <br> (`JSON` or Avro) to a configurable sink  ([Kafka](https://kafka.apache.org/)). |
```

Comment:
Core: "Watches one or multiple tables in a comma-separated list." 
- this is true for both 

Enterprise: "Emits every change to a "watched" row as a record in a configurable format(JSON or Avro) to a configurable sink (Kafka)."
- "Emits every change to a "watched" row as a record" is true for both
- the json/avro format and configurable sink are both true and unique to enterprise

Reply:
Yes, great points! I've incorporated these changes into the PR. On the naming, I think that this is changing soon and there will be more definition. I feel that for now the comparison to core changefeed being available everywhere might be OK and then I can come back in and change soon?